google-datacatalog-looker-connector/tests/google/datacatalog_connectors/looker/prepare/datacatalog_tag_factory_test.py

```diff
@@ -396,9 +400,15 @@ def test_make_tag_for_query_no_model_connection_should_succeed(self):
             'view': 'test-view',
         }
 
+        # yapf: disable
```

Comment:
WDYT about
```
        assembled_metadata = \
            entities.AssembledQueryMetadata(
                serialize.deserialize31(data=json.dumps(query_data),
                                        structure=models.Query),
                'select *', None, None)
```
instead of
```
        # yapf: disable
        assembled_metadata = entities.AssembledQueryMetadata(
            serialize.deserialize31(
                data=json.dumps(query_data),
                structure=models.Query),
            'select *',
            None,
            None)
        # yapf: enable
```
? 🤔

Reply:
between the two I prefer the second (since we are not using yapf), keep a standard and have all parameters aligned. I think in this case the readability is good.

But if you think the first option is better I can change.

Reply:
No need to change, I'm ok with either option. Thanks!